fipy/meshes/abstractMesh.py

```diff
@@ -744,7 +744,7 @@ def facesRight(self):
 
         """
         x = self.faceCenters[0]
-        return x == _madmax(x)
+        return x == self._madmax(x)
```

Comment:
I would prefer a more descriptive function name than ```madmax```. Perhaps ```safemin``` and ```safemax```?

Reply:
I've renamed them `_minmin` and `_maxmax` to denote that they return the minimum of all minima and the maximum of all maxima across processes.